apps/langtonsant/langtons_ant.star

```diff
@@ -0,0 +1,254 @@
+"""
+Applet: Langton's Ant
+Summary: 2-D Cellular Automata
+Description: Automata with simple rules but complex emergent behavior.
+Author: alewando
+"""
+
+load("random.star", "random")
+load("render.star", "render")
+load("schema.star", "schema")
```

Comment:
```suggestion
load("schema.star", "schema")
load("time.star", "time")
```

Reply:
I had similar issues with my game of life app. Creating and reusing a single `Box` pixel for each colour, rather than creating a new `Box` for every pixel made a big difference.

https://github.com/tidbyt/community/pull/584

Reply:
Thank you for the Conway's Life app @dinosaursrarr! Seeing it in the app list was definitely an inspiration for submitting another cellular auotmata. And then seeing your performance solution was brilliant, too!

Reply:
@alewando Inspired by your PR to make this one: https://github.com/tidbyt/community/pull/1240

Reply:
@alewando I went one further and updated the PR with an second app for 1D, three-state CAs.